packages/sdk/src/client.ts

```diff
@@ -461,20 +460,13 @@ export function SoundClient({
 
       return {
         ...release,
-        trackAudio: LazyPromise(() =>
-          soundAPI.audioFromTrack({ trackId: release.track.id }).then((response) => {
```

Comment:
I guess we can remove this query or we keeping for anyone trying to handle presale allow list cases?

Reply:
keeping the query on the graphql side I don't think it hurts, since if we then add the authentication logic, we would want to re-add the track audio query again

Reply:
and also the SoundAPI side it's technically usable on it's own and removing queries just for the sake of removing I don't think it's the best